src/posix/Makefile.am

```
@@ -57,6 +57,7 @@ LIBTOOLFLAGS_COMMON = --preserve-dup-deps
 LDADD_COMMON                                                           = \
     $(top_builddir)/src/posix/platform/libopenthread-posix.a             \
     -lutil                                                               \
+	$(top_builddir)/third_party/tcplp/libtcplp.a                         \
```

nit: use spaces instead of tab
```suggestion
    $(top_builddir)/third_party/tcplp/libtcplp.a                         \
```

---

I've addressed this now in the latest version.